2021-02-06 22:12:05,356 INFO    MainThread:80688 [wandb_setup.py:_flush():70] setting env: {}
2021-02-06 22:12:05,356 INFO    MainThread:80688 [wandb_setup.py:_flush():70] multiprocessing start_methods=spawn
2021-02-06 22:12:05,356 INFO    MainThread:80688 [wandb_setup.py:_flush():70] setting login settings: {}
2021-02-06 22:12:05,356 INFO    MainThread:80688 [wandb_init.py:_log_setup():318] Logging user logs to C:\Users\Juhyeon\Desktop\repository\Proxy-Anchor-CVPR2020\code\wandb\run-20210206_221205-dk96b7h1\logs\debug.log
2021-02-06 22:12:05,356 INFO    MainThread:80688 [wandb_init.py:_log_setup():319] Logging internal logs to C:\Users\Juhyeon\Desktop\repository\Proxy-Anchor-CVPR2020\code\wandb\run-20210206_221205-dk96b7h1\logs\debug-internal.log
2021-02-06 22:12:07,176 INFO    MainThread:80688 [wandb_run.py:_console_start():1399] atexit reg
2021-02-06 22:12:07,177 INFO    MainThread:80688 [wandb_run.py:_redirect():1262] redirect: SettingsConsole.WRAP
2021-02-06 22:12:07,178 INFO    MainThread:80688 [wandb_run.py:_redirect():1297] Wrapping output streams.
2021-02-06 22:12:07,178 INFO    MainThread:80688 [wandb_run.py:_redirect():1313] Redirects installed.
2021-02-06 22:12:07,179 INFO    MainThread:80688 [wandb_run.py:_config_callback():663] config_cb None None {'LOG_DIR': '../logs', 'dataset': 'cars', 'sz_embedding': 512, 'sz_batch': 128, 'nb_epochs': 60, 'gpu_id': 0, 'nb_workers': 0, 'model': 'googlenet', 'loss': 'NPair', 'optimizer': 'adamw', 'lr': 0.0001, 'weight_decay': 0.0001, 'lr_decay_step': 20, 'lr_decay_gamma': 0.5, 'alpha': 32, 'mrg': 0.1, 'IPC': None, 'warm': 1, 'bn_freeze': 1, 'l2_norm': 1, 'remark': ''}
2021-02-06 22:12:09,504 INFO    MainThread:80688 [wandb_run.py:_atexit_cleanup():1369] got exitcode: 1
2021-02-06 22:12:09,505 INFO    MainThread:80688 [wandb_run.py:_restore():1341] restore
2021-02-06 22:12:09,736 INFO    MainThread:80688 [wandb_run.py:_wait_for_finish():1492] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1258
  total_bytes: 1258
}

2021-02-06 22:12:11,739 INFO    MainThread:80688 [wandb_run.py:_wait_for_finish():1492] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7903
  total_bytes: 7903
}

2021-02-06 22:12:12,821 INFO    MainThread:80688 [wandb_run.py:_show_files():1714] logging synced files
2021-02-06 22:12:12,822 INFO    MainThread:80688 [internal.py:handle_exit():156] Internal process exited
